aten/src/ATen/native/Convolution.cpp

```diff
@@ -1454,7 +1454,8 @@ at::Tensor _convolution(
       break;
     case ConvBackend::Empty:
     {
-      auto weight_view = at::_unsafe_view(weight, -1);
+      auto weight_view = weight.is_contiguous() ? at::_unsafe_view(weight, -1) :
```

Comment:
It would be good to add some sort of comment, why this is needed

Reply:
Add comments